airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/state/WorkflowState.java

```diff
@@ -131,6 +132,16 @@ public void setCancelledForReset(final boolean cancelledForReset) {
     this.cancelledForReset = cancelledForReset;
   }
 
+  public void setResetWithScheduling(final boolean resetWithScheduling) {
+    final ChangedStateEvent event = new ChangedStateEvent(
+        StateField.RESET_WITH_SCHEDULING,
+        resetWithScheduling);
+    stateChangedListener.addEvent(id, event);
+    this.resetWithScheduling = resetWithScheduling;
+  }
+
+  // TODO: bmoric -> This is noisy when inpecting the list of event, it should be just a single reset
```

Comment:
can you explain this more?

Reply:
When a reset happens it will register all the events as false. This is noisy when debugging a test since it is not actual events.